### ISSUE TITLE
V7 Node has no ID error when editing a page

### ISSUE THREAD

**Issue by richarth** [umbraco/Umbraco-CMS#4841]
When editing a small number of pages in the back office I receive an error that the node doesn't have an ID attribute.  I've uploaded the full stack trace [here](https://gist.github.com/richarth/bdaebe383c1e4a99f2af9def838701c2).

The site is using Umbraco 7.13.1 and the content has been imported from an Umbraco 6 site using the CMS Import package.

I've checked the XML cache file and the entries for the affected pages do have an ID set in the file.  I've run the data integrity health check which says everything is fine.  I've also tried republishing some of the affected pages and republishing the entire site but this hasn't helped.

**Comment by nul800sebastiaan** [comment]
Hey theree @richarth - This is not something other people experience and I would suspect the import to have done something improper. 

I would recommend you head on over to the forum at https://our.umbraco.com to get some help with this 👍

**Comment by richarth** [comment]
Just in case anybody has this issue in future I managed to identify the cause.

The problem was with a MNTP property that we had added to the document type.  We're migrating an Umbraco 6 site to 7 and the Umbraco 6 site was using . (a dot) as an XPath expression.  We'd therefore set the xpath query as . on the property in Umbraco 7.

It was the dot as the xpath query that was causing Umbraco to error when you try and create/edit a page of that type.  Removing the xpath query stopped Umbraco erroring.
